### PR TITLE
Fix Compile error, and warning in Visaul Studio 2013

### PR DESCRIPTION
**Pull request by DavidSchuldenfrei**

### CODE REVIEW

**Comment by DavidSchuldenfrei** [comment]
@gennadiycivil @nicolastagliani Same as PR 1815, but to branch 1.8.x.

File: googletest/include/gtest/gtest-printers.h [hunk 1]
```diff
@@ -983,7 +983,7 @@ struct TuplePolicy {
 
   template <size_t I>
   static typename AddReference<const typename ::std::tr1::tuple_element<
-      static_cast<int>(I), Tuple>::type>::type
+      I, Tuple>::type>::type
```
Comment (gennadiycivil) [inline googletest/include/gtest/gtest-printers.h]: As I explained in https://github.com/google/googletest/pull/1815 - that change was made for a reason. We can not just roll it back. You may want to consider using macros to only do this when this version of MSVC is used .

**Comment by DavidSchuldenfrei** [comment]
@gennadiycivil Changed as requested.

**Comment by gennadiycivil** [comment]
@DavidSchuldenfrei ping.

**Comment by BrukerJWD** [comment]
Thanks for fixing this! I searched quite some time until I found this here. So for others struggling with this error (which was also described in #1810) I try to point Google to this PR here. Error was:
> error C2770: invalid explicit template argument(s) for 'const ::std::tr1::tuple_element<I,std::tuple<<unnamed-symbol>>>::type testing::internal::TuplePolicy<TupleT>::get(const std::tuple<<unnamed-symbol>> &)'

@DavidSchuldenfrei: Shouldn't we also fix the `tuple_element` a few lines above?

**Pull request merged**